error.go

```diff
@@ -36,3 +36,29 @@ func Value2[T any, U any](out1 T, out2 U, err error) (T, U) {
 func V2[T any, U any](out1 T, out2 U, err error) (T, U) {
 	return Value2(out1, out2, err)
 }
+
+// Value3 panics on error, otherwise returns the first 3 values.
+func Value3[T any, U any, V any](out1 T, out2 U, out3 V, err error) (T, U, V) {
+	if err != nil {
+		panic(err)
+	}
+	return out1, out2, out3
+}
+
+// V2 is short for Value3, which panics on error, otherwise returns the first 3 values.
```

Comment:
typo: V2 should be V3

Reply:
I'm sorry. This should have been caught by automated tools.

Fixed.

Reply:
I thought so too. Surprised golangci-lint did not catch it in its default configuration in the ci. But good thing it didn't because I won't have checked for the other typo so closely.